audformat/core/utils.py

```diff
@@ -529,7 +529,7 @@ def intersect(
         ...         segmented_index(['f1', 'f2'], [0, 1], [1, 2]),
         ...     ]
         ... )
-        MultiIndex([('f1', '0 days', '0 days 00:00:01')],
+        MultiIndex([('f1', '0 days 00:00:00', '0 days 00:00:01')],
```

Comment:
I have no idea why it suddenly prints it differently, but we can see that dtypes are still correct:

```python
index = audformat.utils.intersect([
    audformat.segmented_index(['f1'], [0], [1]),
    audformat.segmented_index(['f1', 'f2'], [0, 1], [1, 2]),
])
index.dtypes
```
```
file              string
start    timedelta64[ns]
end      timedelta64[ns]
dtype: object
```

Reply:
Hehe, yes, you never know why this happens. At least there was no new `pandas` release yesterday.